e2e/prerender/tests/tests.spec.js

```diff
@@ -0,0 +1,121 @@
+// @ts-check
+import { test, expect } from '@playwright/test';
+
+/** @param {import("@playwright/test").BrowserContext} context */
+async function waitForWasm(context) {
+	return new Promise((res) => {
+		context.route(/duckdb-eh\..*?\.wasm$/, async (route) => {
+			await route.abort();
+			res(undefined);
+		});
+	});
+}
```

Comment:
This should go in `e2e/test-utils.js` and accept `page` as an argument (not `context`) to keep consistency with the other existing util function